azure-pipelines.yml

```diff
@@ -47,6 +51,21 @@ jobs:
           ProjectDirectory: 'packages/common'
           Arguments: 'build'
 
+      - script: yarn deploy
+        displayName: 'Deploy Common Package'
+        condition: and(succeeded(), eq(variables.isPR, false))
+        env:
+          HOME: '$(Agent.BuildDirectory)'
+          BRANCH: '$(Build.SourceBranchName)'
+          PULL_REQUEST: '$(isPR)'
+          COMMIT_MESSAGE: '$(Build.SourceVersionMessage)'
+          PACKAGE_LOCATION: 'packages/common'
+          PACKAGE_NAME: 'common'
+          SITE_NAME: 'script-lab-react-storybook'
```

Comment:
do we need this?  We aren't using Storybook any longer...

Reply:
What is the site name that this is deployed to? I still see storybook in our portal

Reply:
I think this is a bit misleading and that we may actually still deploy common to this site, but that the name doesn't make a lot of sense anymore. I dont think this is actually tied to storybook

Reply:
I think Lexa is right. From my understanding the site name we use just happens to have the word storybook. Closing for now, and if we feel like it we can rename all that later